ygen/enumgen_test.go

```diff
@@ -82,6 +172,64 @@ func TestFindEnumSet(t *testing.T) {
 			},
 		},
 		wantSame: true,
+	}, {
+		name: "simple identityref that conflicts",
+		in: map[string]*yang.Entry{
+			"/container/config/identityref-leaf": {
+				Name: "identityref-leaf",
+				Type: &yang.YangType{
+					Name: "identityref",
+					IdentityBase: &yang.Identity{
+						Name: "base-identity",
+						Parent: &yang.Module{
+							Name: "test-module",
+						},
+					},
+				},
+			},
+			"/container/state/identityref-leaf": {
+				Name: "identityref-leaf",
+				Type: &yang.YangType{
+					Name: "identityref",
+					IdentityBase: &yang.Identity{
+						Name: "base-identity",
+						Parent: &yang.Module{
+							Name: "test-module",
+						},
+					},
+				},
+			},
+			"/container/config/identityref-leaf2": {
+				Name: "identityref-leaf",
+				Type: &yang.YangType{
+					Name: "identityref",
+					IdentityBase: &yang.Identity{
+						Name: "baseIdentity",
```

Comment:
Please add a comment here saying that `base-identity` and `baseIdentity` are camel-cased to the same thing -- just so that it's evident to the reader why this is a conflict.

Reply:
I won't do this the next PR will render this test obsolete.